avocado/utils/nvme.py

```diff
@@ -84,9 +84,9 @@ def get_controller_id(controll_name):
     :param controller_name: Name of the controller eg: nvme0
     :rtype: string
     """
-    cmd = f"nvme list-ctrl /dev/{controll_name}"
-    output = process.system_output(cmd, shell=True, ignore_status=True).decode("utf-8")
+    cmd = f"nvme id-ctrl /dev/{controll_name}"
+    output = process.run(cmd, shell=True, ignore_status=True).stdout_text
```

Comment:
`system_output` is not helping?

Reply:
just to reduce the line length I have used this. both will work, I chose this.